### PR TITLE
feat(gitops-wg/README.md): history -> timeline

### PR DESCRIPTION
**Pull request by lloydchang**
link to videos of:
* February: [GitOps Working Group - Overview and Invitation.](https://archive.fosdem.org/2021/schedule/event/gitops_working_group/)
* May: [GitOpsCon EU 2021.](https://www.youtube.com/playlist?list=PLXOML2VBdIo7xEp8Bo9kFB-d6tTlHK5Fk)
* June: [GitOps Days 2021.](https://www.youtube.com/playlist?list=PL9lTuCFNLaD3bglbKKia5ELAoutBupygT)
* June: [GitOps Summit 2021.](https://www.youtube.com/playlist?list=PL2KXbZ9-EY9TRND2YHxordGt8pOw5r45R)

link to:
* October: [GitOpsCon NA 2021.](https://www.youtube.com/playlist?list=PLj6h78yzYM2O_aoiru-ZeyVT0VHdWglfZ)
* October: [Flux GitOps One-Stop Shop Event.](https://fluxcd.io/blog/2021/11/november-2021-update/#gitops-one-stop-shop-event)
* December: [ArgoCon 2021.](https://argoproj.github.io/argocon21/)

align with:
* https://github.com/open-gitops/website/pull/44
* https://github.com/gitops-working-group/gitops-working-group/pull/175

### CODE REVIEW

**Comment by lloydchang** [comment]
Crossposting in [#wg-gitops](https://cloud-native.slack.com/archives/C01G9DEE88M/p1634528778154600) [Slack](https://cloud-native.slack.com/archives/C01G9DEE88M/p1634528778154600) and the [1](https://github.com/open-gitops/website/pull/44), [2](https://github.com/gitops-working-group/gitops-working-group/pull/175), [3](https://github.com/cncf/tag-app-delivery/pull/148) GitHub pull requests…

Thank you!  I empathize that many of you are traveling, resting or recuperating from your tireless efforts for GitOpsCon and KubeCon North America.

👋👀👂✈️🛌😴

Objective:
* My objective is to make it clearer that [#wg-gitops](https://cloud-native.slack.com/archives/C01G9DEE88M/p1634528778154600) try our best to balance various GitOps implementations by featuring videos of demonstrating different implementations,  including Argo CD and Flux

Status:
* Thank you for the reviews and approvals in https://github.com/cncf/tag-app-delivery/pull/148

Question:
* Is there a regular cadence for reviews and approvals of:
* https://github.com/open-gitops/website/pull/44
* and
* https://github.com/cncf/tag-app-delivery/pull/148
?

Thank you everyone 🙂

File: gitops-wg/README.md
Comment (scottrigby) [inline gitops-wg/README.md:44]: @lloydchang my personal pref is to keep the older table style, to more easily see content diffs when updating the table. When aligning like this whenever there's a longer line, the entire table needs to be updated rather than just appending as in the earlier style.

**Review by scottrigby — CHANGES_REQUESTED** [review]
@lloydchang do you know why the git diff is not showing, and it appears to be an entirely new file? Are there perhaps line-ending changes?

File: gitops-wg/README.md
Comment (lloydchang) [reply gitops-wg/README.md:44]: @scottrigby asked:
> Question: do you know why the git diff is not showing, and it appears to be an entirely new file? Are there perhaps line-ending changes?

Why: I renamed the file from `README.MD` to `README.md` to align with [other `README.md` files throughout this repository](https://github.com/cncf/tag-app-delivery/blob/master/README.md), and with [the `README.md` file in the gitops-working-group repository](https://github.com/gitops-working-group/gitops-working-group/blob/main/README.md).

What:  I want to mitigate the minor risk of other people encountering the Developer Experience (DX) with `git add` and `git status` that I experienced.

How: The DX has been:
1. `git add README.md`
2. `git add` failed silently with a `0` exit code
3. `git status` won't display `README.md` because the filename was previously named `README.MD`

Where: I don't know if the previous filename `README.MD` from https://github.com/cncf/tag-app-delivery/pull/146/files was created by a sticky shift key, or sticky caps lock key ⌨️, or by a typo 😅

As an inside joke about this DX — If you don't mind, then I can start calling you by the nicknames of Doc or Dr. @scottrigby, M.D. ... On second thought, I'm just kidding about calling you by nicknames 😄

Comment (scottrigby) [reply gitops-wg/README.md:44]: Oh ha that was prob me. Let's not mix renaming the file into the same PR though, at least not like this, because it makes it nearly impossible to see the diff of what you're actually proposing changing

**Comment by scottrigby** [comment]
🚧 Repo maintainers, please do not merge this PR yet. See https://github.com/cncf/tag-app-delivery/pull/148#discussion_r747817113. The GitOps WG will also be able to sort this out ourselves after https://github.com/cncf/tag-app-delivery/issues/154.
Thanks!

**Comment by scottrigby** [comment]
@lloydchang see #159 which contains your git contribution history and fixes the above issues

**Pull request closed without merging**
